tchannel/tornado/peer.py

```diff
@@ -624,7 +624,7 @@ def send_with_retry(self, request, peer, attempt_times, retry_delay):
                 )
 
                 if not connection:
-                    raise
+                    raise protocol_error
```

Comment:
Looks like this constitutes a breaking change? We'll want to document that in `UPGRADING.rst` and `CHANGES.rst`.